src/main/java/io/rcktapp/api/handler/firehose/FirehoseDb.java

```diff
@@ -45,6 +46,8 @@ public class FirehoseDb extends Db
      */
     protected String includeStreams;
 
+    protected int listDeliveryStreamsLimit = 20;
```

Comment:
I imagine you put this here to make a bigger page then didn't notice that it was unused by the time you finished.